client/src/js/filters/filters.js

```diff
@@ -1,109 +1,111 @@
+// (function (angular) {
+'use strict';
+
+angular.module('bhima.filters')
+  .filter('boolean', function() {
+    return function (input) {
+      return Boolean(Number(input));
+    };
+  })
+
+  // Potential currency filter providing a specified locale 
+  // Ideally would leverage angular currency filter but this does NOT support
+  // passing through individual locale and doesn't expose utility methods 
+  .filter('intlcurrency', [
```

Comment:
Is this filter still necessary?  In your [PR](https://github.com/IMA-WorldHealth/bhima/pull/689) you make no mention of it.  If not, can it be safely removed (all the modules have been migrated, or course)?